examples/api/docs/index.md

```diff
@@ -1,6 +1,4 @@
 ```js
-import "npm:apache-arrow";
-import "npm:parquet-wasm/esm/arrow1.js";
```

Comment:
This was to preload Arrow and parquet-wasm, but we do the preloading automatically now so this isn’t needed.